### PR TITLE
Remove national flags from language selector.

### PR DESCRIPTION
**Pull request by sprrw**
As discussed on #1171, this PR removes national flags from language selector and introduces language change icon instead.

### CODE REVIEW

**Comment by sprrw** [comment]
Related: #966, #1148.

File: components/navigation/navigation.jsx [hunk 1]
```diff
@@ -93,22 +91,22 @@ export default class Navigation extends React.Component {
           <Link
             className="navigation__icon"
             title="GitHub Repository"
-            to="//github.com/webpack/webpack">
+          to="//github.com/webpack/webpack">
```
Comment (skipjack) [inline components/navigation/navigation.jsx]: Can you fix the formatting here?

File: components/navigation/navigation.jsx [hunk 2]
```diff
@@ -93,22 +91,22 @@ export default class Navigation extends React.Component {
...
             <i className="sidecar__icon icon-github" />
           </Link>
 
           <Link
             className="navigation__icon"
             title="See Questions on Stack Overflow"
-            to="//stackoverflow.com/questions/tagged/webpack">
+          to="//stackoverflow.com/questions/tagged/webpack">
```
Comment (skipjack) [inline components/navigation/navigation.jsx]: And here too?

File: components/navigation/navigation.jsx (hunk 1 above)
Comment (sprrw) [inline components/navigation/navigation.jsx]: Yeah, I fixed it. Strange thing is that my editor does favor editorconfig, but for an obscure reason prefers to format those fragments the way they were in the source (and doesn't even let me edit them back). Some misbehaving Atom package I guess.

**Comment by sprrw** [comment]
@skipjack @bebraw I fixed formatting, it looks like it's good to go now.

**Comment by skipjack** [comment]
Awesome work, thanks @sprrw! Will merge as soon as the build passes.

**Pull request merged**